test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -2570,6 +2570,7 @@ BOOST_AUTO_TEST_CASE(storage_location_local_variables)
 				uint[] storage x;
 				uint[] memory y;
 				uint[] memory z;
+				x;y;z;
```

Comment:
I actually think those might not even be necessary because most tests do not require "no warnings".